### DISCUSSION TITLE
iFlight Blitz M10 GPS - Not detecting signal in INAV Configurator 6.1.0

### DISCUSSION THREAD

**Discussion by sgermain06** [iNavFlight/inav#9308] [Q&A]
I was able to configure my MAMBA F722 MK4 using inav-configurator 6.1.0, but for some reason, when I turn on the GPS, it keeps beeping, saying that the hardware is not healthy. I've tried UBLOX and UBLOX7, same problem. I had it configured using Betaflight before and GPS was working just fine. The magnetometer on the M10 seems to be working correctly.

I configured them both (INAV and BF) the same way, using UART3 and 115200 baud, but INAV is still not cooperating. Any ideas?

I'm using the MAMBAF722_2022A firmware, that's available from the firmware upgrade tab.

**Comment by sgermain06** [comment]
Anyone else have issues with this?

**Reply by leopl87** [reply]
Same problem with inav 6.1.1 on matek f405 wing v2. set it uart 3 with a be220 gps/analog vtx/ tbs crossfire rx. Set all auto baudrate and autodetect assistance.
I reflash with 6.0 and work but I try with a open field to test the wing....

**Reply by RadhayRadFPV** [reply]
i am facing similar issue with neutronrc at32 f435 aio (target- NEUTRONRCF435MINI) on inav 7. mag working good but gps not working, showing red gps icon on top and bad hardware health. it was working all good with betaflight but not in inav. were u able to rectify the issue?

**Reply by sensei-hacker** [reply]
@RadhayRadFPV 

Please show a screenshot of your ports tab, and gps tab. Also, which GPS are you using, please?

**Reply by RadhayRadFPV** [reply]
its a foxeer m10q gps
![Screenshot (1)](https://github.com/iNavFlight/inav/assets/159742764/d496b48c-6c4b-4418-a110-0aed19f3035e)
![Screenshot (2)](https://github.com/iNavFlight/inav/assets/159742764/343e8ebe-686a-4a6d-88d9-7e52ec32d537)
![Screenshot (3)](https://github.com/iNavFlight/inav/assets/159742764/74d10319-dfc0-45dc-a2fa-6b47ddb23493)

**Comment by RadhayRadFPV** [comment]
found out the uart 2, uart 3 pad's tx and rx r interchanged in inav, so when i connected tx to tx and rx to rx  gps working fine. is there any way to revert the pads through remapping coz i also want to fly with betaflight without changes in soldering everytime
